<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputWidget</class>
 <widget class="QWidget" name="InputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="RCInput">
      <attribute name="title">
       <string>RC Input</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="advancedPage">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="channelSettings"/>
           </item>
           <item>
            <widget class="QPushButton" name="configurationWizard">
             <property name="text">
              <string>Start Configuration Wizard</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="runCalibration">
             <property name="text">
              <string>Run Calibration</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="wizard">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="wzText">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>90</height>
              </size>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QGraphicsView" name="graphicsView"/>
             </item>
             <item>
              <layout class="QVBoxLayout" name="checkBoxesLayout"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="wzBack">
               <property name="text">
                <string>Back</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="wzNext">
               <property name="text">
                <string>Next</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="wzCancel">
               <property name="text">
                <string>Cancel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Flight Mode Switch Settings</string>
      </attribute>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>10</y>
         <width>201</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Warning: avoid &quot;Manual&quot;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>30</y>
         <width>201</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>for multirotors!</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>445</width>
         <height>155</height>
        </rect>
       </property>
       <property name="title">
        <string>Configure each stabilization mode for each axis</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Roll</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignHCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Pitch</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignHCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Yaw</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignHCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Stabilized1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="fmsSsPos1Roll"/>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="fmsSsPos1Pitch"/>
        </item>
        <item row="1" column="3">
         <widget class="QComboBox" name="fmsSsPos1Yaw"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Stabilized2</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="fmsSsPos2Roll"/>
        </item>
        <item row="2" column="2">
         <widget class="QComboBox" name="fmsSsPos2Pitch"/>
        </item>
        <item row="2" column="3">
         <widget class="QComboBox" name="fmsSsPos2Yaw"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Stabilized3</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="fmsSsPos3Roll"/>
        </item>
        <item row="3" column="2">
         <widget class="QComboBox" name="fmsSsPos3Pitch"/>
        </item>
        <item row="3" column="3">
         <widget class="QComboBox" name="fmsSsPos3Yaw"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>0</y>
         <width>261</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title">
        <string>FlightMode Switch Positions</string>
       </property>
       <widget class="QComboBox" name="fmsModePos2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>55</y>
          <width>151</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="fmsModePos3">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>25</y>
          <width>151</width>
          <height>26</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Select the stabilization mode on this position (manual/stabilized/auto)</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>62</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Pos. 3</string>
        </property>
       </widget>
       <widget class="QComboBox" name="fmsModePos1">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>85</y>
          <width>151</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>62</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Pos. 2</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>62</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Pos. 1</string>
        </property>
       </widget>
       <widget class="QSlider" name="fmsSlider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>20</width>
          <height>81</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>This slider moves when you move the flight mode switch
on your remote. Setup the flightmode channel on the RC Input tab
if you have not done so already.</string>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="pageStep">
         <number>10</number>
        </property>
        <property name="sliderPosition">
         <number>-100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>100</number>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Arming Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Arm airframe using throttle off and:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="armControl">
           <property name="toolTip">
            <string>Indicate the control used for arming the airframe, in addition to setting the throttle to its minimum position. In other terms &quot;Throttle Off&quot;.</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Arming timeout:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="armTimeout">
           <property name="toolTip">
            <string>After the time indicated here, the frame go back to disarmed state.</string>
           </property>
           <property name="maximum">
            <number>64</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>seconds.</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Airframe disarm is done by throttle off and opposite of above combination.</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="inputHelp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../coreplugin/core.qrc">
         <normaloff>:/core/images/helpicon.svg</normaloff>:/core/images/helpicon.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveRCInputToRAM">
       <property name="toolTip">
        <string>Send to OpenPilot but don't write in SD.
Be sure to set the Neutral position on all sliders before sending!</string>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveRCInputToSD">
       <property name="toolTip">
        <string>Be sure to set the Neutral position on all sliders before sending!
Applies and Saves all settings to SD</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fmsSlider</tabstop>
  <tabstop>fmsModePos3</tabstop>
  <tabstop>fmsSsPos3Roll</tabstop>
  <tabstop>fmsSsPos3Pitch</tabstop>
  <tabstop>fmsSsPos3Yaw</tabstop>
  <tabstop>fmsModePos2</tabstop>
  <tabstop>fmsSsPos2Roll</tabstop>
  <tabstop>fmsSsPos2Pitch</tabstop>
  <tabstop>fmsSsPos2Yaw</tabstop>
  <tabstop>fmsModePos1</tabstop>
  <tabstop>fmsSsPos1Roll</tabstop>
  <tabstop>fmsSsPos1Pitch</tabstop>
  <tabstop>fmsSsPos1Yaw</tabstop>
 </tabstops>
 <resources>
  <include location="../coreplugin/core.qrc"/>
 </resources>
 <connections/>
</ui>
